bake/entitlements_test.go

```diff
@@ -180,10 +180,18 @@ func TestValidateEntitlements(t *testing.T) {
 	dir2, err := osutil.GetLongPathName(t.TempDir())
 	require.NoError(t, err)
 
+	// the paths returned by entitlements validation will have symlinks resolved
+	expDir1, err := filepath.EvalSymlinks(dir1)
```

Comment:
In this case I think we can remove `osutil.GetLongPathName` for `dir1` related to https://github.com/docker/buildx/pull/2812#issuecomment-2493632641